### ISSUE TITLE
Terraform 0.12.2 init -force-copy issue

### ISSUE THREAD

**Issue by vasilij-icabbi** [hashicorp/terraform#21719]
### Terraform Version
0.12.2

When I use `terraform init -force-copy` on already existing local back-end, to move it to S3, terraform.tfstate created in .terraform folder has parameter "version": 3, whereas in S3 it has "version": 4. Issue appears when I want to destroy project, I pull state, remove backend configuration and replace state file in .terraform folder with one that I pulled and run destroy. I get  error saying that my state versions are different.

**Comment by vasilij-icabbi** [comment]
The message I get is:

Error: Failed to load state: Terraform 0.12.2 does not support state version 4, please update.

**Comment by jbardin** [comment]
Hi @vasilij-icabbi,

Sorry you're having trouble with this. The `.terraform/` directory is controlled by terraform, and should not be manually edited, and the `terraform.tfstate` file in there is not related to your actual state. 

Can you provide an example of how to reproduce your issue, and the associated logs if possible? Taking a local state and migrating it to s3 works for the simple case, so there must be something different in your configuration.

**Comment by vasilij-icabbi** [comment]
Hello @jbardin 

Thank you for your reply, indeed I do something a bit not standard as you suggested. What I do, is solving chicken-egg problem by having S3 backend as resource in Terraform. Thing is, that I did some wrong steps on my side and therefore this issue is not an issue anymore.

As you correctly mentioned state file in there is not an actual state file, basically I was copying actual state file there, but instead I had to remove the backend state file.

It solved on my end, ticket can be closed.
